Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -868,5 +867,11 @@ public void MarkAsDirty()
         {
             LastChanged = DateTime.UtcNow;
         }
+
+        public bool CheckSystemValidity()
```

Comment:
please extend this test:
`\Tests.Unit.Core.ApplicationServices\Model\ItSystemUsageTest.cs`
so that it proves all cases:
- Valid if date fields are valid and active is not set
- Valid if active is valid and invalid according to date fields
- Invalid if both conditions are false
- Valid if both conditions are true